recipes-extended/onl/onl_git.bb

```diff
@@ -10,6 +10,7 @@ LIC_FILES_CHECKSUM += "file://LICENSE;beginline=14;md5=7e6d108802170df125adb4f45
 
 SRCREV_onl ?= "dd61cc1792290a459c394c5b7bdf4d4a27b6f651"
 URI_ONL ?= "git://github.com/opencomputeproject/OpenNetworkLinux.git;protocol=https;branch=master"
+PV = "2022-04-08+git${SRCPV}"
```

Comment:
Can you add a comment above this line to let the next person to update PV know when and how to update the string?

Reply:
Added a comment.